pkg/broker/handler/pool/fanout_test.go

```diff
@@ -200,6 +200,7 @@ func TestFanoutSyncPoolE2E(t *testing.T) {
 	})
 
 	t.Run("event with delivery timeout was sent to retry queue", func(t *testing.T) {
+		t.SkipNow()
```

Comment:
Add a comment linking to an issue.

Reply:
After some more digging I was able to discover the issue. I added context cancellation to pubsub receive in the SDK (https://github.com/cloudevents/sdk-go/pull/469) which caused goroutines in this test which previously ignored the context timeout and blocked forever to correctly return with an error. This SDK issue was covering up a bug which would have otherwise been detected by this test.

#1087 should fix this issue, I will wait to merge this PR until after the fix is merged.